backstop/tests/table.html

```diff
@@ -3793,32 +4184,112 @@ <h2>Empty state</h2>
     </section>
 
     <script>
-      var selectAllFn = function (selectAllId, tableId) {
-        var element = document.getElementById(selectAllId);
-        element.indeterminate = true;
-        element.onclick = function () {
-          var checkboxes = document.querySelectorAll(`#${tableId} .iui-row input[type="checkbox"]:not(:disabled)`);
-          for (var checkbox of checkboxes) {
-            checkbox.checked = this.checked;
-          }
-          element.indeterminate = false;
-          element.checked = this.checked;
-        };
+      const setTableHeaderCheckbox = (table) => {
```

Comment:
This makes row to get `iui-selected` when checkbox is clicked. Also all header checkboxes now selects table body rows.